assets/js/src/subscribers/importExport/import/step_method_selection/method_paste.jsx

```diff
@@ -1,19 +1,61 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import MailPoet from 'mailpoet';
+import ReactStringReplace from 'react-string-replace';
+
+const kbLink = 'http://docs.mailpoet.com/article/126-importing-subscribers-with-csv-files'
```

Comment:
This link redirects to https://kb.mailpoet.com/article/126-importing-subscribers-with-csv-files
so maybe we could update it